lucida/djinntonic/dig/Makefile

```diff
@@ -23,7 +23,7 @@ $(TARGET): $(OBJECTS)
 	$(CXX) -Wall $(CXXFLAGS) -c $< -o $@
 
 start_server: 
-	gnome-terminal -x bash -c "./DIGServer $(filter-out $@,$(MAKECMDGOALS)); read -n1"
+	./DIGServer
 
 start_test: 
 	gnome-terminal -x bash -c "./run-test.sh $(filter-out $@,$(MAKECMDGOALS)); read -n1"
```

Comment:
There currently isn't a automated way to test all of the servers so when someone wants to run tests they have to go to each individual microservice folder and run start_test. In this situation it may be necessary to have a new terminal open as it won't be necessary to keep it running after the test has been run unlike a server.